dll/win32/kernel32/kernel32_vista/runonce.c

```diff
@@ -0,0 +1,80 @@
+/*
+ * PROJECT:         ReactOS system libraries
+ * COPYRIGHT:       See COPYING in the top level directory
+ * PURPOSE:         Wrappers for the NT One-Time Initialization Implementation
+ * PROGRAMMER:      Ratin Gao <ratin@knsoft.org>
+ */
+
+#include "k32_vista.h"
+
+NTSYSAPI
+DWORD
+WINAPI
+RtlRunOnceBeginInitialize(
+    RTL_RUN_ONCE *once,
+    ULONG flags,
+    void **context);
+
+NTSYSAPI
+DWORD
+WINAPI
+RtlRunOnceExecuteOnce(
+    RTL_RUN_ONCE *once,
+    PRTL_RUN_ONCE_INIT_FN func,
+    void *param,
+    void **context);
```

Comment:
Use `VOID`?

Reply:
`RtlRunOnceXxx` we currently have, are taken from wine, those declarations are incompatible with Windows SDK and our XDK, because they use `DWORD`, `WINAPI`, and `void` you saw, and no SAL annotations.

I cannot move them to our PSDK unless modify them (may be in a separate PR in the future?), so I paste them here unmodified, and I suggest we keep it unmodified here.

Reply:
Those functions are already declared in xdk/rtlfuncs.h , and so, either those latter declarations can be copied where necessary in the NDK, or the corresponding generated header (XDK generates headers...) be included.

Reply:
Committed, I copied those declarations come from `xdk/rtlfuncs.h`